### ISSUE TITLE
[Bug]: Incorrect message in lyrics view for instrumental songs

### ISSUE THREAD

**Issue by pgalhardo** [ciderapp/Cider#923]
### Describe the Bug

An incorrect message is being displayed in lyrics view when instrumental songs are playing.

![Capture](https://user-images.githubusercontent.com/44944781/164916346-840aa48b-39c7-449a-9dfb-898aededfb5a.JPG)

The term "term.noLyrics" ("Loading... / Lyrics not found./ Instrumental.") seems to have been created for this purpose, but for some reason this string is being displayed instead.

### Steps to Reproduce

1. Find and play an instrumental song
2. Open the lyrics view
3. The lyrics view shows an incorrect message

### Environment Information

- OS: Windows 10
- App Version and/or Commit ID: 2822fa6c54db2af8f8d2900cd50a3165f06da786

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by noahdominic** [comment]
The bug is also present on Linux, v1.4.3.1843 from flatpak. 

It seems to come from either `loadMXM()` or `getMXMSubs()` in  [`vueapp.js`](https://github.com/ciderapp/Cider/blob/2822fa6c54db2af8f8d2900cd50a3165f06da786/src/renderer/main/vueapp.js#L2671). Not sure which specifically. In any case, the app tries to get lyrics from musixmatch.com, fails, and falls back to some `catch(e)` to get lyrics from qq.com. The text on Cider matches the one on the qq.com website as well (see the qq.com entry for [_Mafia_](https://y.qq.com/n/ryqq/songDetail/003kdwAn1tphkj) and a few others) so I can be fairly certain this is what was happening, but I'm honestly not familiar enough with Cider to suggest what the appropriate fixes would be. (I just installed it yesterday.)

**Comment by UnbreakCode** [comment]
sometimes it also replaces the lyrics from musixmatch with the lyrics from qq.com.

**Comment by noahdominic** [comment]
I've been looking at dev tools and musixmatch DOES return a response object for both songs with lyrics and pure instrumentals. with an important difference, specifically in `message.body.macro_calls["track.subtitles.get"].message.body`

Here's the response object for _Redlight_ by Swedish House Mafia & Sting, a track with lyrics: 
![image](https://user-images.githubusercontent.com/22269703/165201451-4a258547-f420-4936-9e7c-13a5bd2bac44.png)

On the other hand, here's the response object for _Mafia_ by Swedish House Mafia, a purely instrumental track: 
![image](https://user-images.githubusercontent.com/22269703/165201610-e5869aa6-2b01-4fd5-86d7-8365aa4e86c7.png)

Note how `message.body.macro_calls["track.subtitles.get"].message.body` is empty for instrumentals.  This becomes a problem on line [2756](https://github.com/ciderapp/Cider/blob/2822fa6c54db2af8f8d2900cd50a3165f06da786/src/renderer/main/vueapp.js#L2756) 
```javascript
...
lrcfile = jsonResponse["message"]["body"]["macro_calls"]["track.subtitles.get"]["message"]["body"]["subtitle_list"][0]["subtitle"]["subtitle_body"];
...
```
when something inside `message.body.macro_calls["track.subtitles.get"].message.body` is assigned to `lrcfile`.  This variable is used in line [2756](https://github.com/ciderapp/Cider/blob/2822fa6c54db2af8f8d2900cd50a3165f06da786/src/renderer/main/vueapp.js#L2756) where it is checked if it is an empty string
```javascript
...
if (lrcfile == "") {
    app.loadQQLyrics();
    // app.loadAMLyrics()
} else {
    ...
```
Since `message.body.macro_calls["track.subtitles.get"].message.body.subtitle_list[0].subtitle.subtitle_body` doesn't exist for instrumentals, `lrcfile` will remain unchanged from its init value of an empty string (line [2677](https://github.com/ciderapp/Cider/blob/2822fa6c54db2af8f8d2900cd50a3165f06da786/src/renderer/main/vueapp.js#L2677)) and `loadQQLyrics()` is subsequently called.

This check should be modified. Maybe see whether it's an instrumental first, since a flag exists for that? Like this:
```diff
...
- if (lrcfile == "") {
+ if (lrcfile == "" && jsonResponse["message"]["body"]["macro_calls"]["track.subtitles.get"]["message"]["header"]["instrumental"] == 0) {
    app.loadQQLyrics();
    // app.loadAMLyrics()
 } 
...
```

Though, I don't think this is a complete solution, because the code in the `else` block requires manipulation of the contents of `lrcfile`, and I'm not sure how well it will handle empty strings.  Or maybe there's some other solution I'm not seeing.

**Comment by pgalhardo** [comment]
Fixed by 130816a1f08b36f83adcef9afa549bd41a8f49ad
